content/hardware/01.mkr/02.shields/mkr-therm-shield/product.md

```diff
@@ -7,4 +7,4 @@ certifications: [CE]
 
 The MKR Therm Shield is a great addon for MKR family boards, and allows you to connect high quality thermocouplers, and can calculate temperatures from -200 °C to +700 °C. An ideal solution to use for ovens, freezers, smokers and similar environments.
 
-To use this shield, you can refer to the documentation of the [MKR therm library](https://www.arduino.cc/reference/en/libraries/arduino_mkrtherm/).
\ No newline at end of file
+To use this shield, you can refer to the documentation of the [Arduino MKR THERM library](https://www.arduino.cc/reference/en/libraries/arduino_mkrtherm/).
```

Comment:
```suggestion
To use this shield, you can refer to the documentation of the [Arduino_MKRTHERM library](https://www.arduino.cc/reference/en/libraries/arduino_mkrtherm/).
```

Actually the library name is [Arduino_MKRTHERM](https://github.com/arduino-libraries/Arduino_MKRTHERM/blob/master/library.properties#L1) (by Arduino).